### ISSUE TITLE
Footer

### ISSUE THREAD

**Issue by ffMathy** [dogger/app.dogger.io#649]
Terms, blog, privacy, contact.

Email should be in the footer. Cloudflare spam protection should be turned on.

**Comment by heybereket** [comment]
Any design you would like it in? Text-align center? right? left?

**Comment by ffMathy** [comment]
@bereketsemagn is that because you're considering contributing? 😍

**Comment by heybereket** [comment]
Yeah, I am considering @ffMathy 
I'd be more than happy to contribute/be apart of the organization :)

Would that be possible?

**Comment by ffMathy** [comment]
You can always contribute if you want! Let me know if you need help! <3

**Comment by heybereket** [comment]
Thank you!

**Comment by ffMathy** [comment]
No, thank YOU! Also, you look quite young? Awesome that you got started early.

If the contributions are good, I can write you a recommendation on LinkedIn. Perhaps that can help in your career!

**Comment by heybereket** [comment]
Yeah, I have a big passion for programming. Thank you! I'll start working on the README file and go deeper towards adding the footer, etc. Looking forward to it! Gave you a follow on GitHub, you seem very friendly :)
